### PR TITLE
[Diggy] Armor items in market

### PR DESCRIPTION
**Pull request by SimonFlapse**
Also spread out the item unlock further.

See https://docs.google.com/spreadsheets/d/1legWZSHFDd6b3IO42wWE_oSZJtkIGuVPRVquwQ3r0J8/edit#gid=0

### CODE REVIEW

File: map_gen/Diggy/Feature/MarketExchange.lua [hunk 1]
```diff
@@ -142,12 +142,21 @@ local function update_market_contents(market)
 
             local name = unlockable.prototype.name
             local price = unlockable.prototype.price
-            print('Mining Foreman: New wares at the market! Come get your ' .. name .. ' for only ' .. price .. ' ' .. config.currency_item .. '!')
+            print(type(price))
+            if type(price) == 'number' then
+                print('Mining Foreman: New wares at the market! Come get your ' .. name .. ' for only ' .. price .. ' ' .. config.currency_item .. '!')
```
Comment (iltar) [inline map_gen/Diggy/Feature/MarketExchange.lua]: lets remove this case and the config option for the currency, it adds nothing imo

File: map_gen/Diggy/Feature/MarketExchange.lua [hunk 2]
```diff
@@ -142,12 +142,21 @@ local function update_market_contents(market)
 
             local name = unlockable.prototype.name
             local price = unlockable.prototype.price
-            print('Mining Foreman: New wares at the market! Come get your ' .. name .. ' for only ' .. price .. ' ' .. config.currency_item .. '!')
+            print(type(price))
```
Comment (Valansch) [inline map_gen/Diggy/Feature/MarketExchange.lua]: Is this print left here intentionally? Seems like a debug artifact to me.

Comment (SimonFlarup) [reply map_gen/Diggy/Feature/MarketExchange.lua]: Oh yeah sorry...

Comment (SimonFlapse) [reply map_gen/Diggy/Feature/MarketExchange.lua]: Yeah that's not supposed to be there

File: map_gen/Diggy/Feature/MarketExchange.lua (hunk 1 above)
Comment (SimonFlapse) [reply map_gen/Diggy/Feature/MarketExchange.lua]: Should I make it only print out one line for all items? Instead of one per item?

**Comment by SimonFlapse** [comment]
I'm adjusting the items right now

File: map_gen/Diggy/Config.lua [hunk 3]
```diff
@@ -284,27 +284,39 @@ local Config = {
...
+                    {level = 46, price = 250, name = 'uranium-rounds-magazine'},
+                    {level = 58, price = {{"stone", 1500}, {"coin", 25}}, name = 'rocket-launcher'},
+                    {level = 58, price = {{"stone", 500}, {"coin", 5}}, name = 'rocket'},
+                    {level = 66, price = {{"stone", 1000}, {"coin", 10}}, name = 'explosive-rocket'},
+                    {level = 73, price = {{"stone", 1000}, {"coin", 200}}, name = 'satellite'},
+                    {level = 100, price = {{"stone", 5000}, {"coin", 9999}}, name = 'atomic-bomb'},
                 }
             ),
-
+--mk1 power armour, lasers, batteries, roboports, sheilds, maybe mk2
```
Comment (iltar) [inline map_gen/Diggy/Config.lua]: what's this comment about?

Comment (SimonFlapse) [reply map_gen/Diggy/Config.lua]: :( Needed something to remind me what to add

**Comment by iltar** [comment]
Alright, lets see if it works!

**Pull request merged**